src/api/app/views/shared/_package_branch_form.html.haml

```diff
@@ -14,6 +14,7 @@
   %label{ for: 'current_revision' } Stay on current revision
```

Comment:
I think a problem here is also the naming in the UI as it shouldn't be ``current revision`` but ``this revision``, no?